modules/mobile/app/models/mobile/v0/adapters/disability_rating.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'common/models/base'
+require 'date'
+
+module Mobile
+  module V0
+    module Adapters
+      class DisabilityRating
+        def parse(data)
+          Mobile::V0::Rating.new(
+            id: 0,
+            combined_disability_rating: data['combined_disability_rating'],
+            individual_ratings: individual_ratings(data)
+          )
+        end
+
+        private
+
+        def individual_ratings(response)
+          response['individual_ratings'].map do |rating|
+            Mobile::V0::IndividualRating.new(
+              decision: rating['decision'],
+              effective_date: parse_date(rating['effective_date']),
+              rating_percentage: rating['rating_percentage']&.to_i(),
```

Comment:
Sorry to continue being super nitpicky, but I think your IDE may have autocompleted the parens here.

Reply:
all good. fixed